### DISCUSSION TITLE
Installer should Run as Administrator natively

### DISCUSSION THREAD

**Discussion by slacksmatt** [setvisible/DownZemAll#195] [Ideas]
When I tried to install the software it didn't prompts Administrator privilege.
Especially when you tried to install the software into different destination, it failed to install the software without any notices and only write shortcuts in the start menu

**Comment by setvisible** [comment]
Good point. This should be fixed.
FYI I configured the NSIS install script to install with lowest user rights (not admin), because the "queue.json" file - which is dedicated to an user - is installed in the same directory. It's a dirty setup I concede.
